synapse/tools/rstorm.py

```diff
@@ -56,49 +203,42 @@ async def genStormRst(path, debug=False):
 
             text = line.split('::', 1)[1].strip()
 
-            core = context.get('cortex')
-            if core is None:
-                mesg = 'No cortex set.  Use .. storm-cortex::'
-                raise s_exc.NoSuchVar(mesg=mesg)
+            core = reqCore()
 
-            opts = context.get('opts')
-            await core.callStorm(text, opts=opts)
+            soutp = StormOutput(core, context, stormopts=context.get('storm-opts'))
+            await soutp.runCmdLine(text)
             continue
 
-        if line.startswith('.. storm::'):
+        if line.startswith('.. storm-pkg::'):
+            # load a package into the cortex from a directory
 
-            text = line.split('::', 1)[1].strip()
+            yamlpath = line.split('::', 1)[1].strip()
```

Comment:
It definitely feels like there is a relative path issue here with embedding paths into RST.

Reply:
We should fail here if the path does not exist.

Reply:
ack